README.md

```diff
@@ -29,7 +28,7 @@ Some of the features:
 
 Requirements:
 
-* Erlang/OTP version R16B or later
+* Erlang/OTP version 19 or later
```

Comment:
@zuiderkwast it surprises me a little that you are willing to go this step since you always seemed to take special pride in being compatible all the way back to R16 :sweat_smile: But I'm really in favor of this step. In fact, it would allow us to remove the backporting of `gen_server:close` which we have somewhere in there, since that appeared in OTP 18.

Reply:
Haha, yeah, after many years we can make a small bump.

We could probably go as fast as OTP 22 or something, but it annoys me that tools like rebar3 only support the two latest major versions. That's really to little. We should at least cover what's included in the oldest LTS distros that are still not yet end-of-life. It used to be CentOS but now I guess it's AlmaLinux or something.

Reply:
It's `gen_server:stop`, right?

Reply:
> It's `gen_server:stop`, right?

Yeah, that's what I meant :)